onnxruntime/core/providers/cpu/tensor/col2im.cc

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/cpu/tensor/col2im.h"
+#include "core/util/math_cpuonly.h"
+
+
+namespace onnxruntime {
+
+#define REGISTER_COL2IM_TYPED_KERNEL(OP_TYPE, VERSION, TYPE, KERNEL_CLASS)                  \
+  ONNX_CPU_OPERATOR_TYPED_KERNEL(                                                           \
```

Comment:
Fixed the domain, but I am still confused where to put the implementation, because it seems there was a change in policy where to add new ops to don't affect minimal build binary size.

Adding to `onnxruntime/contrib_ops/cpu` seems to increase size of minimal build binary size, as mentioned here https://onnxruntime.ai/docs/reference/operators/add-custom-op.html#contrib-ops

Reply:
The binary size is affected regardless of where the implementation is located, if the kernel is included in the build.

Are you referring to this?
https://github.com/microsoft/onnxruntime/blob/97180a10563bf41fd12d927843682878510a83c6/docs/reference/operators/add-custom-op.md?plain=1#L58

Perhaps that is suggesting the use of a custom op instead of adding a contrib op to avoid adding to ORT's binary size, but I suppose that doesn't apply here.

See also:
https://github.com/microsoft/onnxruntime/blob/97180a10563bf41fd12d927843682878510a83c6/docs/reference/operators/add-custom-op.md?plain=1#L81-L83